examples/grafana/grafana-ocp.yaml

```diff
@@ -26,51 +53,609 @@ objects:
     host: "${ROUTE_URL}"
     to:
       name: grafana-ocp
+    tls:
+      termination: Reencrypt
 - apiVersion: v1
   kind: Service
   metadata:
     name: grafana-ocp
+    annotations:
+      prometheus.io/scrape: "true"
```

Comment:
Does grafana expose prometheus metrics?

Reply:
yes, you can query mostly anything in prometheus.